<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSetup</class>
 <widget class="QDialog" name="CameraSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1841</width>
    <height>1151</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>931</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>1021</x>
     <y>51</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>曝光时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gain">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>150</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>增益：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_focus">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>260</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>焦距：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>白平衡温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_area">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>500</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>聚焦区域：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>600</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>600</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>600</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_h">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>600</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>height:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_time">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>20</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_gain">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>130</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_focus">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>240</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_temp">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>400</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>2800</number>
   </property>
   <property name="maximum">
    <number>6500</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_open_auto">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开自动白平衡</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_show_box">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>700</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示聚焦框</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_time">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_gain">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>150</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_focus">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_reset">
   <property name="geometry">
    <rect>
     <x>1590</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_temp">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>410</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_area">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>490</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_x">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>600</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_h">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>600</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_w">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>600</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>600</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_backl">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>750</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/btexit.png);</string>
   </property>
   <property name="text">
    <string>   返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Setup_CameraSetup_lbCam">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>801</y>
     <width>931</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pButton_open_view">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>打开预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButton_stop_view">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>停止预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButton_open_light">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>打开灯光</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButton_close_light">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>关闭灯光</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButton_auto_focus">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>自动对焦</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
